ml/src/main/scala/org/apache/spark/ml/h2o/param/H2OMOJOModelParams.scala

```diff
@@ -32,56 +32,33 @@ trait H2OMOJOModelParams extends DeprecatableParams {
   //
   // Param definitions
   //
-  private final val labelCol: Param[String] = new Param[String](this, "labelCol", "Label column name")
-  private final val featuresCols: StringArrayParam = new StringArrayParam(this, "featuresCols", "Name of feature columns")
+  private val labelCol: Param[String] = new Param[String](this, "labelCol", "Label column name")
+  protected final val featuresCols: StringArrayParam = new StringArrayParam(this, "featuresCols", "Name of feature columns")
   private final val outputCol: Param[String] = new Param[String](this, "outputCol", "Column where predictions are created")
   private final val convertUnknownCategoricalLevelsToNa = new BooleanParam(this,
          "convertUnknownCategoricalLevelsToNa",
          "Convert unknown categorical levels to NA during predictions")
   //
   // Default values
   //
-  setDefault(labelCol -> "label")
   setDefault(featuresCols -> Array.empty[String])
   setDefault(outputCol -> "prediction_output")
   setDefault(convertUnknownCategoricalLevelsToNa -> false)
 
   //
   // Getters
   //
-  /** @group getParam */
-  @DeprecatedMethod("getLabelCol")
-  def getPredictionsCol(): String = getLabelCol()
```

Comment:
remove getter for label col

Reply:
Shouldn't we do such changes only in major releases?